### ISSUE TITLE
[RFE] Auto remove leading space in Search box

### ISSUE THREAD

**Issue by tjyang** [NagiosEnterprises/nagioscore#681]
* WHAT: Existing NavBarSearchItem class doesn't remove input string with leading space characters.
* WHY: to avoid confusing when doing search. 
* HOW:
    * Modify NavBarSearchItem php code to remove leading spaces. 
    * side.php
```
 <form method="get" action="<?php echo $cfg["cgi_base_url"];?>/status.cgi" target="<?php echo $link_target;?>">
                        <fieldset>
                                <legend>Search:</legend>
                                <input type='hidden' name='navbarsearch' value='1'>
                                <input type='text' name='host' size='15' class="NavBarSearchItem">
                        </fieldset>
                </form>

```
 
* Docs:
    * [Google Search has clue on using trim php function ](https://www.google.com/search?q=php+form+remove+leading+spaces&rlz=1C1GCEV_en&oq=php+form+remove+lea&aqs=chrome.2.69i57j33l5.9829j0j7&sourceid=chrome&ie=UTF-8)

**Comment by ericloyd** [comment]
Thanks.  We've been asking for this since Nagios XI 5.0

**Comment by Madlohe** [comment]
Hi @tjyang, thanks for reporting this! I'm pushing a fix for this to maint, would appreciate if you could verify that it's working as expected for you.

@ericloyd, do you know if there's another open issue for this? I didn't see anything but I wouldn't want to leave the issue open if it's fixed now.

**Comment by ericloyd** [comment]
It's probably on Nagios XI Customer Forum post, not here.  No worries. :-)

**Comment by tjyang** [comment]
@Madlohe , I would  love to see and test the PR in my test server.

**Comment by tjyang** [comment]
@Madlohe 

I patched 4.4.4:/cgi/status.c and test the search. As shown below  strings of "    locahost" and "  localhost    " 

![image](https://user-images.githubusercontent.com/378638/62979448-76393100-bde9-11e9-99e7-2c3069317810.png)

**Comment by Madlohe** [comment]
@tjyang The commit above (https://github.com/NagiosEnterprises/nagioscore/commit/b26e3cb31a75a5a7b13e3a357ea7f9fcb6b7ea37) is in the maint branch. It should go out with the next release
